tests/utils.ts

```diff
@@ -0,0 +1,42 @@
+import * as request from 'supertest';
+import { getConnection } from 'typeorm';
+import * as winston from 'winston';
+
+import createServer from '../src/server';
+import { createConnection } from '../src/db';
+
+export const setup = createConnection;
+export const clearDb = () => getConnection().synchronize(true);
+
+function buildNoopLogger() {
+  const logger = winston.createLogger({
+    transports: [new winston.transports.Console()],
+    exitOnError: false,
+  });
+  logger.transports.pop();
+
+  return logger;
+}
+
+const app = createServer({
+  logger: buildNoopLogger(),
+  stream: { write: () => {} },
+});
+
+const setDefaultHeaders = (httpRequest: request.Test) => {
+  httpRequest.set('Accept', 'application/json');
+  httpRequest.set('Origin', 'http://localhost:3000');
```

Comment:
This won't be fine in the future. Use a config variable.

Reply:
I should remove it and add it if we ever really need it